### PR TITLE
[docs] Add info about nightmare

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Use one of [built-in reporters](https://devexpress.github.io/testcafe/documentat
 
 ### Straightforward Continuous Integration
 
-TestCafe is easy to set up on popular Continuous Integration platforms as it allows you to test against various browsers: local, remote or cloud (e.g., [Sauce Labs](https://saucelabs.com/)).
+TestCafe is easy to set up on popular Continuous Integration platforms as it allows you to test against various browsers: local, remote, cloud (e.g., [Sauce Labs](https://saucelabs.com/)) or headless (e.g. [Nightmare](http://www.nightmarejs.org)).
```
Comment (inikulin) [inline README.md]: I guess it's better to give links to our plugins here

Comment (inikulin) [inline README.md]: Or even better to recipies

File: docs/articles/documentation/extending-testcafe/browser-provider-plugin/index.md [hunk 2]
```diff
@@ -18,7 +18,9 @@ This topic contains the following sections.
 * [Publishing the Provider to npm](#publishing-the-provider-to-npm)
 * [Specifying a Browser for a Test Task](#specifying-a-browser-for-a-test-task)
 
-> To run tests on [Sause Labs](https://saucelabs.com), you can use the existing [testcafe-browser-provider-saucelabs](https://github.com/DevExpress/testcafe-browser-provider-saucelabs/) plugin.
+> To run tests on [Sause Labs](https://saucelabs.com) or [Nightmare](https://github.com/segmentio/nightmare) browsers,
```
Comment (inikulin) [inline docs/articles/documentation/extending-testcafe/browser-provider-plugin/index.md]: Replace Nightmare with `headless` here

File: README.md (hunk 1 above)
Comment (AlexanderMoskovkin) [inline README.md]: > Or even better to recipies

Do you mean to create a recipe for this?

Comment (inikulin) [inline README.md]: I mean we already have recipe for saucelabs - link it. For nightmare - just give link to plugin

Comment (AlexanderMoskovkin) [inline README.md]: got it

**Pull request merged**